### ISSUE TITLE
Vetur: status quo, development, sponsorship and direction

### ISSUE THREAD

**Issue by octref** [vuejs/vetur#1982]
Sorry for the silence in the past few months. I was dealing with the coronavirus situation (my family is in Wuhan) and [left US for China](https://blog.matsu.io/on-leaving).

Now my life is more settled down, I'm picking up development again. I finished triaging all issues, and I plan to review a few PRs to make a release in the next few days.

I wrote a longer blog post as well [Vetur: status quo, development, sponsorship and direction.
](https://blog.matsu.io/vetur-status-quo-development-sponsorship-and-direction)

I'm also starting a sponsorship: https://github.com/sponsors/octref. You can read more about why in my [blog post](https://blog.matsu.io/vetur-status-quo-development-sponsorship-and-direction#sponsorship).

Finally, quite a few people have shown interest in contributing to Vetur. I don't have much experience organizing efforts around OSS projects, but I'll give it a try. I'll start by updating the contribution docs and isolating feature areas that can be independently contributed to. In the following days I'll prioritize reviewing PRs instead of doing development by myself.

Meanwhile, if you are interested in contributing to Vetur, of if you have great ideas how to make Vetur easier to contribute to, please respond here. Ideally I hope I can delegate a specific area completely to another contributor, such as [interpolation transformation](https://github.com/vuejs/vetur/blob/master/server/src/services/typescriptService/transformTemplate.ts) to @ktsn.

**Comment by yoyo930021** [comment]
I use Vue and TypeScript in my work,
and study LSP and TypeScript Compiler API in my free time.
I should be able to help reply issues, TypeScript parts, and performance problems.

**Comment by KytoSai** [comment]
I hope you can priotitize the focus on the multi root support feature.

Because i see it being voted the most popular in the roadmap v1.0. There are also a lot of issues related to this feature. (https://github.com/vuejs/vetur/issues/873)

I see some PR to solve this problems. Hope you can take the time to look over

Ex: https://github.com/vuejs/vetur/pull/1928 

Thank you ! ^^

**Comment by tiagoroldao** [comment]
I have started contributing just recently, and while it was the first time contributing to a VSCode plugin, I found it a really smooth experience. Kudos on how cleanly this project has been run so far (and I guess kudos on the VSCode team for their part).

Would love to get a bit more involved from now on!

**Comment by morkro** [comment]
I would like to offer support as well. At the time of writing this, there are 19 open pull requests of which some seem to solve existing issues but there is no communication on what happens with them. Right now it seems a mystery to an outsider what the status with those is, why they're not getting merged (as it looks like they fix known issues), and how do handle stale requests. It feels discouraging to contribute if there are so many open and unhandled requests.

I am happy to get involved and help out with the community part (or any other way) of maintaining this great library :)

**Comment by yoyo930021** [comment]
@morkro 
The author @octref will handle small PRs first, and then bigger.
Give him some time. Thank you.

**Comment by octref** [comment]
> At the time of writing this, there are 19 open pull requests of which some seem to solve existing issues but there is no communication on what happens with them.

That's because the ones being responded to were already closed. I previously had 2x that PRs that I haven't reviewed.
When developer time > PR review time, you'd see each PR being responded to.
When developer time < PR review time, which is my case, I review a PR, do some simple cleanup and merge them immediately, unless the PR needs major changes.

**Comment by morkro** [comment]
I understand, sorry I realise my previous message comes off negatively. It wasn't meant like that 🙂

Is there any way to support with that?

**Comment by octref** [comment]
Hey @morkro, sorry if I came across as dismissing. I know I'm behind on reviewing PR.

This week I have improved the CI a lot, so for small changes I can easily merge them.
Larger ones take a lot of testing to ensure they wouldn't break other existing functionalities. I didn't have enough time to address them. And there's the perf concern — sometimes I have to do a release with them without fully understanding the perf impact, and be ready to do hotfixes in the next few days.

I'm starting to formalize the contribution process in the wiki: https://github.com/vuejs/vetur/wiki/Triaging-Issues
Later I'll add more wiki for contributing code / helping with PR review as well.
Thanks for being willing to help out 🤝
